crates/interpreter/src/valid.rs

```diff
@@ -764,17 +804,50 @@ impl<'a, 'm> Expr<'a, 'm> {
         Ok(())
     }
 
-    fn br_label(&self, l: LabelIdx) -> Result<ResultType<'m>, Error> {
+    fn stitch(&mut self, branch: &SideTableBranch) {
```

Comment:
If `SideTableBranch` is `Copy`, we should not take it by reference.
```suggestion
    fn stitch(&mut self, branch: SideTableBranch) {
```